docs/hoon/rune/tis/wut.md

```diff
@@ -0,0 +1,40 @@
+---
+navhome: /docs/
+next: true
+sort: 13
+title: =? "tiswut"
+---
+
+# `=? "tiswut"`
+
+`[$tswt p/wing q/hoon r/hoon s/hoon]`: conditionally change one leg in the subject.
+
+## Expands to
+
+```
+=. a ?. =(b c) a c
```

Comment:
Should this be `=.  p  ?.  =(q r)  p  s` to match the structure in the header?

Reply:
yes

Reply:
Actually it needed more changes than that so I made them---how does this look?